imagepicker/build.gradle

```diff
@@ -10,8 +10,6 @@ android {
     defaultConfig {
         minSdkVersion 16
         targetSdkVersion 31
-        versionCode 1
```

Comment:
Why these changes? I know they don't matter though, any reason to remove them>?

Reply:
For some reason when I updated my AS to Bumblebee this happened